examples/todo-mvc-vue/src/todos.machine.ts

```diff
@@ -0,0 +1,127 @@
+import { createMachine, spawn, ActorRef } from 'xstate';
+// @ts-expect-error
```

Comment:
What's the error here?

Reply:
`uuid-v4` has no typescript definitions and `@types/uuid-v4` does not exist...
I used the same package from your example but could use a different package that has TS definitions :)

Reply:
Changed uuid-v4 to nanoid in ba4c936 to remove this error :)